dotnet3.5/S3/DeleteBucketExample/DeleteBucket/DeleteBucket.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX - License - Identifier: Apache - 2.0
+// snippet-start:[S3.dotnet35.DeleteBucket]
+
+using Amazon;
+using Amazon.S3;
+using System;
+using System.Threading.Tasks;
+
+namespace DeleteBucket
+{
+    class DeleteBucket
+    {
+        // This example shows how to delete an existing and empty bucket.
```

Comment:
"an existing and empty" -> "an existing empty"

Reply:
I suppose. Removed and.